### DISCUSSION TITLE
Removing borders of DayNames

### DISCUSSION THREAD

**Discussion by korijamal** [nhn/tui.calendar#1356] [Q&A]
## Version

v2.1.3
## Test Environment

Chrome, MacOs, React, Typescipt
## Current Behavior

![Screenshot 2023-01-19 at 9 59 01 AM](https://user-images.githubusercontent.com/81694120/213331059-9c5d18b8-4065-4f74-986d-184a506d792f.png)

```js
// Write example code
```
<Calendar
                  ref={calendarRef}
                  height={height}
                  view="month"
                  isReadOnly={true}
                  calendars={calendars}
                  schedules={schedules}
                  usageStatistics={false}
                  theme={{ "common.border": "none", "month.dayName.borderRight": "none" }}
                  month={{
                    daynames: ["일", "월", "화", "수", "목", "금", "토"],
                    scheduleFilter: function (schedule) {
                      console.log("schedule:" + schedule);
                      return schedule.isVisible;
                    },
                    visibleWeeksCount: 1,
                  }}
                />

## Expected Behavior

I want it to look clean with out any borders!

**Comment by korijamal** [comment]
I tried these so far 
theme= {{"week.dayName.borderRight": "none"}} 
theme= {{"week.borderRight": "none"}}
theme= {{"week.border": "none"}}

**Comment by adhrinae** [comment]
It's the `borderLeft`, not `borderRight`.

https://github.com/nhn/tui.calendar/blob/main/docs/ko/apis/theme.md#month-dayname

I'm not sure why there are borders out there because the default value is `none`.

Why don't you open the browser devtools and inspect elements?
